repos/system_upgrade/el7toel8/actors/checkchrony/actor.py

```diff
@@ -0,0 +1,53 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.libraries.common.reporting import report_generic
+from leapp.libraries.stdlib import CalledProcessError, run
+
+
+class CheckChrony(Actor):
+    """
+    Check for incompatible changes in chrony configuration.
+
+    Warn that the default chrony configuration in RHEL8 uses the leapsectz
+    directive.
+    """
+
+    name = 'check_chrony'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        installed_packages = set()
+
+        signed_rpms = self.consume(InstalledRedHatSignedRPM)
+        for rpm_pkgs in signed_rpms:
+            for pkg in rpm_pkgs.items:
+                installed_packages.add(pkg.name)
+
+        if 'chrony' not in installed_packages:
+            report_generic(
```

Comment:
Hmm I wonder what's the purpose/value of this report.
Imagine someone would want to read the report and is told about all the packages that are not installed. 

Couldn't this be solved by a log message instead?

Reply:
Ok. Makes sense.